create-presigned-download-url/src/main/resources/download_file.feature

```diff
@@ -0,0 +1,32 @@
+Feature: Retrieve download URL for a Publication File
+
+  Scenario: A User retrieves a download URL for a Publication File
+    Given the User is the owner of the Publication
+    When they set the Accept header to "application/json"
+    And they set the Authentication header to a Bearer token with their credentials
+    And they request POST /download/{identifier}/files/{fileIdentifier}/generate
```

Comment:
It should be GET. Also the this scenario is not easily runnable but since we are not running them we can fix it later.